aten/src/ATen/cpu/vec/sve/vec_int.h

```diff
@@ -42,6 +42,15 @@ public:
   operator svint##bit##_t() const {                                                                     \
     return values;                                                                                      \
   }                                                                                                     \
+  template <uint64_t mask>                                                                                      \
+  static Vectorized<int##bit##_t> blend(const Vectorized<int##bit##_t>& a, const Vectorized<int##bit##_t>& b) { \
```

Comment:
Very unrelated, but mixing preprocessor with templates is weird. Why it's not a template of 2 parameters that could be specialized later?